src/solution/server/controllers/todoController.js

```diff
@@ -0,0 +1,105 @@
+// Define your endpoints here (this is your "controller file")
```

Comment:
please change returned status codes and make sure your client is handling the errors as it should